riverdriver/riversql/internal/dbsqlc/sqlc.yaml

```diff
@@ -0,0 +1,24 @@
+version: "2"
+sql:
+  - engine: "postgresql"
+    queries:
+      - ../../../riverpgxv5/internal/dbsqlc/river_migration.sql
+    schema:
+      - ../../../riverpgxv5/internal/dbsqlc/river_migration.sql
```

Comment:
Is the thought that in general we should be able to share queries between drivers and let sqlc abstract away the differences? More generated code to execute or compile, but less code to actually maintain?

After reading your comment about how each driver would need to implement the full set of query methods I had some concerns about the amount of duplication we'd end up with, but if we can share the underlying schema and query definitions then that should help considerably.